tests/test_event.py

```diff
@@ -46,66 +36,84 @@ def login_user(self):
         }
         return self.client().post('/api/auth/login', data=user_data)
 
-    def test_api_get_all_events(self):
-        """Test API can get all events without token passed (GET request)."""
+    def get_access_token(self):
+        """register and login a user to get an access token"""
         self.register_user()
         result = self.login_user()
         access_token = json.loads(result.data.decode())['access_token']
+        return access_token
 
+    def create_event_cartegory(self):
+        """This helper method helps create categories."""
+        access_token = self.get_access_token()
+        category_data = {
+            "category_name": "Sports"
+        }
+        res = self.client().post(
+            '/api/category',
+            headers=dict(Authorization="Bearer " + access_token),
+            data=category_data)
+
+        return res
+
+    def event_creation(self):
```

Comment:
according to your docstring it is test method..it is advisable to write it as `test_event_creation` so that nose will be able to run that method.

Reply:
I specified it as a helper method not a test